apps/explorer/src/components/common/Button/index.tsx

```diff
@@ -1,28 +1,9 @@
 import React from 'react'
 
-import styled, { css } from 'styled-components/macro'
-import { ThemeValue, variants } from 'styled-theming'
```

Comment:
Awesome! You got rid of `styled-theming` dependency, please delete it from package.json as well